vitrina/datasets/models.py

```diff
@@ -841,6 +841,39 @@ def get_resource_managers_queryset(self) -> QuerySet["Dataset"]:
             .distinct()
         )
 
+    def get_resource_managers_queryset_specific_role(
+        self, include_information_system: bool = False, include_open_data: bool = False
+    ) -> QuerySet["Dataset"]:
+        datasets_ids = {self.id}
+        organization_ids = {self.organization_id}
+
+        for parent_dataset in self.get_ancestors().only("pk", "organization_id"):
+            datasets_ids.add(parent_dataset.pk)
+            organization_ids.add(parent_dataset.organization_id)
+
+        base_q = (
```

Comment:
Let's avoid one letter names:

```suggestion
        base_queryset = (
```